engine/src/main/java/org/terasology/world/block/internal/BlockManagerImpl.java

```diff
@@ -252,7 +254,11 @@ public BlockFamily getBlockFamily(BlockUri uri) {
                         block.setId(getNextId());
                     }
                     registerFamily(newFamily.get());
-                } finally {
+                }
+                catch (Exception ex){
```

Comment:
What exceptions can be thrown? Can we be more specific here?

Reply:
I'll add a comment, but didn't like how the old implementation just skipped the family

Reply:
Would it be possible to catch a more specific exception type rather than `Exception`?

Reply:
the block family implementation from this point is unknown. It can basically fail in a lot of unexpected ways that are unforeseeable. since the family type can also be implemented on the module end.